lib/engine/game/g_18_zoo/round/operating.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/operating'
+require_relative '../../../step/buy_train'
+
+module Engine
+  module Game
+    module G18ZOO
+      module Round
+        class Operating < Engine::Round::Operating
+          def after_setup
+            super
+
+            @game.corporations.each do |corporation|
+              corporation.all_abilities
```

Comment:
one liner

corporation.all_albities.each do |ability|  

next if ...

Reply:
done in 5d6145b5